GLMakie/src/drawing_primitives.jl

```diff
@@ -280,8 +281,15 @@ function draw_atomic(screen::Screen, scene::Scene, @nospecialize(x::Lines))
         data = Dict{Symbol, Any}(gl_attributes)
 
         positions = handle_view(x[1], data)
+
         transform_func = transform_func_obs(x)
 
+        # Tweak things for px_per_unit
+        resolution = pop!(data, :resolution)
+        px_per_unit = data[:px_per_unit]
+        data[:resolution] = map((ppu, res) -> ppu .* res, px_per_unit, resolution)
```

Comment:
Should we move this to `connect_camera!`? This effectively makes the `resolution` in shaders the true pixel resolution rather than the (potentially) scaled user resolution. Would require one or two tweaks in the scatter shader, not really simplifying or complicating anything...

Reply:
Yeah sounds good! Would also be nice to document this a bit more... I'm pretty lost, where we use which unit/resolution at this point :D I also have the suspicion, that WGLMakie does it a bit different, but comes to the same result..

Reply:
Ok, how should I change this here? I assume we want to avoid small commits, so should I make a pr to this branch? Or just push one commit directly and hope nothing else needs changing as a result?

Reply:
I think a PR against this branch that we squash merge would be best!